cloud/deployment/deployment.go

```diff
@@ -37,6 +37,7 @@ var (
 	ErrWrongEnforceInput         = errors.New("the input to the `--enforce-cicd` flag is invalid. Make sure to use either 'enable' or 'disable'")
 	ErrInvalidResourceRequest    = errors.New("invalid resource request")
 	ErrNotADevelopmentDeployment = errors.New("the Deployment specified is not a development Deployment")
+	errNoDeploymentInWSMsg       = "No Deployments found in workspace %s"
```

Comment:
```suggestion
	errMsgNoDeploymentInWorkspace = "No Deployments found in workspace %s"
```

Reply:
I would not create the local variable for it.

We use this error message in [several places](https://github.com/search?q=repo%3Aastronomer%2Fastro-cli+%22No+Deployments+found+in+workspace%22&type=code). Or we make it `ErrMsg...` and use it everywhere even inside tests.

Reply:
This isn't actually an error though @feluelle, it's the message for the error. I would use something like the following probably? https://play.golang.com/p/J_NfrQ-xFno

```suggestion
	ErrNoDeploymentInWS = "No Deployments found in workspace"
```

Reply:
I think the `err` prefix creates a slight bit of confusion, as we generally use it to define actual error types, so I will drop that and simply use `NoDeploymentInWSMsg` as constant to better represent that it's a message string